sql/moz-fx-data-shared-prod/search_derived/search_revenue_levers_daily_v1/query.sql

```diff
@@ -0,0 +1,283 @@
+WITH
+-- Google Desktop (search + DAU)
+desktop_data_google AS (
+  SELECT
+    submission_date,
+    IF(lower(channel) LIKE '%esr%', 'esr', 'personal') AS channel,
+    IF(country = 'US', 'US', 'RoW') AS country,
+    count(DISTINCT client_id) AS dau,
+    count(
+      DISTINCT IF(sap > 0 AND normalized_engine = 'Google', client_id, NULL)
+    ) AS dau_engaged_w_sap,
+    sum(IF(normalized_engine = 'Google', sap, 0)) AS sap,
+    sum(IF(normalized_engine = 'Google', tagged_sap, 0)) AS tagged_sap,
+    sum(IF(normalized_engine = 'Google', tagged_follow_on, 0)) AS tagged_follow_on,
+    sum(IF(normalized_engine = 'Google', search_with_ads, 0)) AS search_with_ads,
+    sum(IF(normalized_engine = 'Google', ad_click, 0)) AS ad_click
+  FROM
+    `mozdata.search.search_clients_engines_sources_daily`
+  WHERE
+    submission_date = @submission_date
+    AND country NOT IN ('RU', 'UA', 'TR', 'BY', 'KZ', 'CN')
+  GROUP BY
+    1,
```

Comment:
I'd rather use named columns for readability and maintainability (here and in other places in this PR).